Main/Levels/Level.cs

```diff
@@ -92,7 +95,11 @@ public struct LevelMetadata {
         }
         public LevelMetadata(Level level, string[] meta, string name) : this(name, meta,
             level.objects.FindAll(obj => obj.character != LevelObject.holdChar).Count,
-            level.objects.FindAll(obj => obj.character != LevelObject.speedChar).Select(obj => obj.offset).ToList(), level.speeds) { }
+            level.objects.FindAll(obj => obj.character != LevelObject.speedChar).Select(obj => obj.offset).ToList(), level.speeds)
+        {
+            initialOffsetMS = meta.Length > 5 ? int.Parse(meta[5]) : 0;
```

Comment:
This is also not needed, since it's already done in the parent constructor

Reply:
How could I rewrite this to make the value be set in the parent constructor?

Reply:
You already did that, the parent constructor is the constructor above this one, you can just move the logger.Info there and delete this